app/javascript/client/checkout/Checkout.tsx

```diff
@@ -56,20 +124,23 @@ interface CheckoutFormData {
 }
 
 const InternalCheckout: FC<CheckoutProps> = ({ history, unitPrice, cart, subtotal }) => {
+
+    // Comment in the line below once you can add stuff to the cart
+    // if (cart.length === 0) history.push('/home');
```

Comment:
This is just to redirect the user to the home page if there's nothing in the cart, since they wouldn't need the checkout page in that case